src/ariel-os/Cargo.toml

```diff
@@ -64,6 +64,11 @@ coap = ["dep:ariel-os-coap", "random"]
 ## Enables applications to set up CoAP server handlers.
 ## See [`coap::coap_run()`].
 coap-server = ["coap", "ariel-os-coap/coap-server"]
+# Plain forwarded features that are not documented as features but just as laze
+# modules, because while those here work without any extra help from laze, most
+# later ones will likely need some build system help.
```

Comment:
Examples of that build system help are "laze checks whether a file is present, decides which to select, passes it in through the environment, and creates one if none is present"; even for those here, we could have some build system interaction like a yellow blinking "hey that's not secure" (for both, actually).